scheduler/scheduler/start_server.go

```diff
@@ -0,0 +1,102 @@
+package main
```

Comment:
think server.go would be a better name

Reply:
I thought that would be confusing with scheduler/api/server.go and scheduler/server directory.

Since this file is not an object definition - its main feature is the StartServer function (which is not a member function for an object), using the action_name (start_server.go) makes more sense to me.